lib/tasks/hosts.rake

```diff
@@ -36,6 +36,7 @@ namespace :hosts do
       end
     end
 
+    # TODO: Replace with either just out of sync or origin specific interval
```

Comment:
I think here we should also respect the interval that is relevant for the host, if hosts have reports for more systems, I'd pick the longest interval